pom.xml

```diff
@@ -769,7 +769,67 @@
 
     </plugins>
     <pluginManagement>
-      <plugins>
+	      <plugins>
```

Comment:
@erdemedeiros I would appreciate some of your comments here. It seems that this is only for eclipse, so I'm ok with this PR.